code/go/0chain.net/smartcontract/zcnsc/burn.go

```diff
@@ -15,7 +17,7 @@ import (
 func (zcn *ZCNSmartContract) Burn(
 	trans *transaction.Transaction,
 	inputData []byte,
-	balances cstate.StateContextI,
+	ctx cstate.StateContextI,
```

Comment:
ctx is usually a name convention for context.Context, better use different name here